common/protos/fpc.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto3";
```

Comment:
quick thought: shouldn't we create here an fpc sub-directory and add our own stuff there? At least i would have expected that we also move the current google/fabric protobufs into common (including a common build procedure) and then we more or less would need an fpc subdir?

Reply:
I was thinking the same! so we do it!